modules/dhcp_common/server.rb

```diff
@@ -29,6 +29,11 @@ def initialize(name, managed_subnets, subnet_service, free_ips_service = nil)
                          end
     end
 
+    def validate_supported_addresses_ip(ip)
+      # Not implemented. Method overloading for validation.
+      ip
+    end
```

Comment:
Apologies for leading you into this, please make the default implementation to validate IPv4 or IPv6.